app/Entities/Post.php

```diff
@@ -57,6 +57,6 @@ public function setReplyTo(?string $value = null)
 
     public function isMarkedAsDeleted(): bool
     {
-        return $this->attributes['marked_as_deleted'] !== null;
+        return $this->attributes['marked_as_deleted'] ?? null !== null;
```

Comment:
Always `null`. Why this change? If `marked_as_deleted` may not be initialized, then is better?
```php
protected $attributes = [
        'marked_as_deleted' => null
];
```
Although the date cast should apply null itself..

Reply:
This is probably just a workaround for an unmigrated database? At least it seems like it.

Reply:
That's what I figured, too. Pretty sure I ran migrations at that point again, but we can remove this check for now.